lib/bundle/commands/exec.rb

```diff
@@ -98,10 +99,6 @@ def run(*args, global: false, file: nil, subcommand: "")
           ENV.prepend_path "PATH", Pathname.new(dep_root)/"shims"
         end
 
-        # Setup pkg-config, if present, to help locate packages
-        pkgconfig = Formulary.factory("pkg-config")
-        ENV.prepend_path "PATH", pkgconfig.opt_bin.to_s if pkgconfig.any_version_installed?
```

Comment:
This is needed on Linux, since we don't have a `pkg-config` shim there.

Reply:
Good idea, done!